src/main/webapp/script.js

```diff
@@ -6,7 +6,8 @@
  */
 
 function initBody() {
-  checkPermissions();
+    fetch('/data').then(console.log('Works'));
```

Comment:
I think this isn't needed anymore as isn't it calling the test servlet?
```suggestion
```

Reply:
You are right! :D